modules/nf-core/picard/splitsambynumberofreads/main.nf

```diff
@@ -0,0 +1,93 @@
+process PICARD_SPLITSAMBYNUMBEROFREADS {
+    tag "${meta.id}"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/picard:3.4.0--hdfd78af_0' :
+        'biocontainers/picard:3.4.0--hdfd78af_0'}"
+
+    input:
+    tuple val(meta), path(bam)
```

Comment:
```suggestion
    tuple val(meta), path(input)
```
If it can take cram files in, we generally use input as the variable instead of bam